code/obj/storage/secure_crates.dm

```diff
@@ -84,7 +84,8 @@
 		grenades
 			name = "special grenades crate"
 			spawn_contents = list(/obj/item/storage/box/QM_grenadekit_security = 2,\
-			/obj/item/storage/box/QM_grenadekit_experimentalweapons)
+			/obj/item/storage/box/QM_grenadekit_experimentalweapons,\
+			/obj/item/storage/box/stun_landmines)
```

Comment:
skip the QM crate part I think

Reply:
I didn't change them, that's just their type path :shelterfrog:
![image](https://user-images.githubusercontent.com/41448081/140620775-04cdc967-effb-47d9-af9f-3ce9ee29ea3d.png)